Lombiq.NodeJs.Extensions/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "nodejs-extensions",
+  "version": "0.0.1",
+  "private": true,
+  "devDependencies": {
+    "autoprefixer": "^10.4.2",
+    "clean-css-cli": "^5.5.2",
+    "nodemon": "^2.0.15",
+    "npm-run-all": "^4.1.5",
+    "postcss": "^8.4.7",
+    "postcss-cli": "^9.1.0",
+    "rimraf": "^3.0.2",
+    "sass": "^1.49.8",
+    "stylelint": "^14.5.1",
+    "stylelint-config-standard-scss": "^3.0.0"
+  },
+  "scripts": {
+    "echo-env": "npm run env",
+    "build:styles": "SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _build:styles",
```

Comment:
How this works in GE is that if you have linter errors, the SCSS compilation still happens, the CSS is produced (and you get a build warning in MSBuild). It's important to keep this behavior, because during development you can choose to ignore linter violations during quick development, you just need to fix them until the end (i.e. the same as for .NET analyzer violations).